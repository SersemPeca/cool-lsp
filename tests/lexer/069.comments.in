(* hey -- this is not fine *) "hello"
-- (* this is also not fine
"what now?" *)
