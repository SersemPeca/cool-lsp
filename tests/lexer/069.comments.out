#1 STR_CONST "hello"
#3 STR_CONST "what now?"
#3 ERROR: Unmatched *)
